pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/tissue/TissueSMIDDao.java

```diff
@@ -48,15 +44,17 @@ public class TissueSMIDDao {
                     + "LEFT JOIN ddp_participant as p ON (p.participant_id = ins.participant_id) "
                     + "LEFT JOIN ddp_instance as realm ON (realm.ddp_instance_id = p.ddp_instance_id) "
                     + "WHERE realm.instance_name = ?";
+    private static final Logger logger = LoggerFactory.getLogger(TissueSMIDDao.class);
 
-    public String getTypeForName(String type) {
```

Comment:
This method was moved down

Reply:
Not intentionally, it was moved because I wrote a new method on top of it and removed another method  (`createNewSMIDForTissueWithValue`) entirely from this file so git kinda lost it and _thinks_ it was moved and renamed. I think its because they share some lines of code (like creating the transaction) and I'm not sure how to avoid that.